taccsite_cms/static/site_cms/css/src/_imports/trumps/s-drop-cap.css

```
@@ -1,5 +1,15 @@
-@import url('../tools/x-drop-cap.css');
+@supports (initial-letter: 3){
+  .s-drop-cap > p:first-of-type::first-letter {
+    padding-right: 1em; /* approx: `1em` of normal text */
```

This seems **exactly** `1em` of normal text now. 😀

```suggestion
    padding-right: 1em;
```